enlighten/SPCFileWriter.py

```diff
@@ -0,0 +1,672 @@
+# See the following refernces for details on the .spc file format implementation
```

Comment:
One small note.  I won't often ask this, but...could you prepend our MIT license header atop this?  I suspect this file may get copied about a bit.